experiments/tagging/jetclassexperiment.py

```diff
@@ -207,27 +208,29 @@ def _evaluate_single(self, loader, title, mode, step=None):
         )  # unweighted mean of AUCs across classes
         if mode == "eval":
             LOGGER.info(f"The ovo mean AUC is\t\t{metrics['auc_ovo']:.5f}")
-        fpr_list, tpr_list = [], []
-        for i in range(len(self.class_names)):
-            fpr, tpr, _ = roc_curve(labels_true == i, labels_predict[:, i])
-            fpr_list.append(fpr)
-            tpr_list.append(tpr)
-
+        
         # 1/epsB at fixed epsS
-        def get_rej(epsS, class_idx):
-            background_eff_fn = interp1d(tpr_list[class_idx], fpr_list[class_idx])
+        def get_rej(epsS, tpr, fpr):
+            background_eff_fn = interp1d(tpr, fpr)
             return 1 / background_eff_fn(epsS)
 
         class_rej_dict = [None, 0.5, 0.5, 0.5, 0.5, 0.99, 0.5, 0.995, 0.5, 0.5]
 
-        for i, rej in enumerate(class_rej_dict):
-            if rej is None:
-                continue
-            rej_string = str(rej).replace(".", "")
-            metrics[f"rej{rej_string}_{i}"] = get_rej(rej, i)
+        for i in range(len(self.class_names)-1):
```

Comment:
Using `range(1, len(self.class_names))` would save you a lot of `+1` in other places ;)